### DISCUSSION TITLE
[IntelliSense] It doesn't work with Elm programming language

### DISCUSSION THREAD

**Discussion by vitalije** [tailwindlabs/tailwindcss#11494] [Ideas]
When editing class attributes in Elm intellisense is not showing. I've added  `'elm': 'html'`  to the languages map and it does show up if I type `class="..."`, but this is not valid syntax in Elm. In Elm one usually adds class attribute using something like:
```elm
import Html.Attributes as HA
myClass : HA.Attribute
myClass =
    HA.class "...."
```
There is no `=` sign between word `class` and a string literal.

I would be glad to make a PR for this feature, but I am really new to the VSCode and I've no idea even where to look. So, if someone direct me to the right place or give me some clue, I would probably be able to solve this issue.

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Comment by vitalije** [comment]
@wongjn thanks. It really solved my problem.

This is what I've added to the `settings.json` file. (in case someone else needs it too)
```json
    "tailwindCSS.experimental": {
        "classRegex" : [
            ["(class\\s+\"[^\"]*?\")", "class\\s+\"([^\"]*?)\""]
        ]
    },
```
